src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -191,8 +192,8 @@ public static class RSessionEvaluationCommands {
             return evaluation.EvaluateAsync<ulong>(script, REvaluationKind.Normal);
         }
 
-        public static Task<ulong> ExportToPdfAsync(this IRExpressionEvaluator evaluation, Guid deviceId, Guid plotId, string outputFilePath, double widthInInches, double heightInInches) {
-            string script = Invariant($"rtvs:::export_to_pdf({deviceId.ToString().ToRStringLiteral()}, {plotId.ToString().ToRStringLiteral()}, {widthInInches}, {heightInInches})");
+        public static Task<ulong> ExportToPdfAsync(this IRExpressionEvaluator evaluation, Guid deviceId, Guid plotId, ExportPdfParameters pdfParams) {
```

Comment:
Since it is just a helper extension, I'd rather pass all parameters as they are, without intermediate `ExportPdfParameters`. Also, you need to pass `CancellationToken` for it.